pkg/proc/native/ptrace_linux.go

```diff
@@ -82,3 +87,15 @@ func PtraceGetRegset(tid int) (regset linutil.AMD64Xstate, err error) {
 	err = linutil.AMD64XstateRead(regset.Xsave, false, &regset)
 	return
 }
+// PtraceGetRegset returns floating point registers of the specified thread
+// using PTRACE.
+// See arm64_linux_fetch_inferior_registers in gdb/arm64-linux-nat.c.html
+// and arm64_supply_xsave in gdb/arm64-tdep.c.html
+func PtraceGetRegset_arm64(tid int) (regset linutil.ARM64PtraceFpRegs, err error) {
```

Comment:
Create two files ptrace_linux_amd64.go and ptrace_linux_arm64.go. Copy the existing PtraceGetRegset into ptrace_linux_amd64.go and copy this code to ptrace_linux_arm64.go, remove the _arm64 suffix from PtraceGetRegset_arm64.